codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -646,9 +647,10 @@ private LastHttpContent readTrailingHeaders(ByteBuf buffer) {
                         b.append(currentLastPos)
                          .append(lineTrimmed);
                         current.set(lastPos, b.toString());
-                    } else {
-                        // Content-Length, Transfer-Encoding, or Trailer
                     }
+                    //else {
+                    // Content-Length, Transfer-Encoding, or Trailer
```

Comment:
The intention is not to hold up the PR but as the code was modified it is a good opportunity to get input/justifications or things that smell fishy. Opening another issue sgtm (all good @jasontedor)